packages/react/README.md

```diff
@@ -0,0 +1,72 @@
+<div align="center">
+  <p>
+    <img alt="Gadget logo" src="https://raw.githubusercontent.com/gadget-inc/js-clients/main/docs/assets/gadget-logo.png" />
+  </p>
+  <p>
+    <a href="">
+      <img alt="GitHub CI status" src="https://badgen.net/github/checks/gadget-inc/js-clients/main/Test?label=CI" />
+    </a>
+    <a href="https://www.npmjs.com/package/@gadgetinc/react">
+      <img alt="npm version" src="https://badgen.net/npm/dw/@gadgetinc/react?color=4148f2" />
+    </a>
+  </p>
+  <p>
+    <strong>
+      React hooks for use with application APIs built using the <a href="https://gadget.dev">Gadget</a> platform.
+    </strong>
+  </p>
+</div>
+
+## Features
+
+1. An **auto-suggested** `select` option for all hooks, showing all of the available fields on various models
+2. A **typed** and **hydrated** result based on any given `select` option
+
+This library wraps [urql](https://github.com/FormidableLabs/urql) to communicate with a Gadget-generated API, so it benefits from all of
+the same [features](https://github.com/FormidableLabs/urql/blob/main/README.md#-features) as urql.
+
+## Installation
+
+```
+yarn add @gadgetinc/react react
+# or
+npm install --save @gadgetinc/react react
+```
+
+## Example usage
+
+```typescript
+import { useAction, useFindMany } from "@gadgetinc/react";
+import React from "react";
+import { Client } from "@gadget-client/my-gadget-app";
+
+const [_, deleteWidget] = useAction(Client.widget.delete);
```

Comment:
I think you need an instance of the api, no? Or can you really just use the prototype like this? Also, I feel like it might freak people out to see hooks used at the top level of a module as opposed to inside a React component, can I suggest adding a wrapper component?